packages/@unimodules/core/unimodules-core.gradle

```diff
@@ -2,7 +2,9 @@ class UnimodulesPlugin implements Plugin<Project> {
     void apply(Project project) {
         project.configurations.configureEach {
             if (it.getName().startsWith('test') && project.file('./src/test').exists()) {
-                project.dependencies.add(it.getName(), project.project(':unimodules-test-core'))
+                if (project.findProject(':unimodules-test-core')) {
```

Comment:
Looks like this `if` can be put a few lines higher, before `configureEach`, right? 🤔